docs/modules/reference/pages/how-to-build-with-docker.adoc

```diff
@@ -1,29 +1,31 @@
+
 = Build with Docker
 
+This section describes how to build {page-component-title} with Docker.
+
 == Simple build
 
-Helm can be compiled from source with a Docker build image.
+{page-component-title} can be compiled from source with a Docker build image:
 
+. Check out the repository, compile the code, and install the plugin:
++
 [source, console]
-----
 docker build -t myopennmshelmimage .
-----
-
-will checkout the source from GitHub URL of the project, compile and installs the plugin.
-The result is a runnable image which can be executed with
 
+. Run the image:
++
 [source, console]
-----
 docker run -p 3000:3000 -t myopennmshelmimage
-----
 
-Log in on `\http://your-ip:3000` with login admin:admin and go to Plugins and enable the Helm app.
+. Log in to your {page-component-title} instance with the default admin credentials (`admin`/`admin`).
+. Navigate to *Plugins* and enable the *OpenNMS Helm* plugin.
 
-IMPORTANT: The plugin directory location is changed from `/var/lib/grafana/plugins` to `/opt/grafana/plugins` because the base grafana image defines the `/var/lib/grafana` as a _VOLUME_ which can't be changed.
+IMPORTANT: The Grafana plugin directory location is changed from `/var/lib/grafana/plugins` to `/opt/grafana/plugins`.
```

Comment:
```suggestion
IMPORTANT: The Grafana plugin directory location changes from `/var/lib/grafana/plugins` to `/opt/grafana/plugins`.
```

Reply:
I think I want to check the tense and context on this - is it changed because of the steps the user takes, or because they're using Docker?